_posts/2023-02-20-example-mapping-in-practice.markdown

```diff
@@ -0,0 +1,179 @@
+
+The team get about asking questions aroundf the discounts and limit:
+![Example Map](/img/blog/examplemapping/questions.png)
+
+The questions lead to examples being specified, and a new rule emerges which needs examples to help explain:
+
+![Example Map](/img/blog/examplemapping/examplemap.png)
+
+To recap the process. The Product Owner brings a new User Story to the session, presents to the team with a brief explanation and then the facilitator sets to work . I like to run my sessions as follows:
+
+1. Allow the Product Owner to describe the User Story including the how and why
+2. Set a timer for five minutes quiet time and allow the team to raise questions ready for the Product Owner
+3. During the quiet time the Product Owner may start creating Rules on the blue cards
+4. Once quiet time is over I go through the question cards and get the team member who raised it to ask the Product Owner the question
+5. As questions are answered, new rules are defined and then clarified using Examples as a whole team exercise
+6. The board is continually refined as questions are answered, new rules written and examples provided
+7. To ensure that we have something solid to start on, I try to keep focus on one Rule at a time to ensure that sufficient examples are required and continually ask "Dev team, are you happy you can start building this" and "Product Owner, are you happy with this outcome"
+
+At the end of the session the goal is to at least have a starting point for the User Story development and a plan to follow up any additional sessions requried.
+
+<br />
+
+# The Opposition
+
+There's obviouslt many ways to specify and communicate requirements, so lets stack Example Mapping up against the opposition.
+
+## The Func Spec
+
+![Example Mapping](/img/blog/examplemapping/funcspec.jpg)
+
+Ahhh the old faithful! What can go wrong? Well...let me tell you...
+As 'Engineer Liason' [Tom Smykowski](https://www.imdb.com/title/tt0151804/characters/nm0726223) told us in the Movie Office Space _"I deal with the god damn customers so the engineers don't have to. I have people skills; I am good at dealing with people. Can't you understand that? What the hell is wrong with you people?"_
+
+So according to Tom, having the engineers talk directly with the users is just not right, so instead a specialist role exists. Sounds simple right? But here is what really happens:
+
+- The BA talks to 'The Business', converts their business speak into tech requirements and places them in a nice document
+- The developer picks up the func spec and codes their interpretation
+- The tester then picks up the spec and tests the code to the letter of the document
+- The developer and tester get into a massive argument
+- The BA gets involved and tells them that they are both wrong
+- 'The Business' then come along and tell them all that the spec does not reflect what they wanted
+
+And all that is based on the fact that the developer even reads the spec.
```

Comment:
This sentence doesn't read well for me - hard to make out which point you are making?

The developer doesn't (often?) read the spec?

Reply:
That was the point yes, removed the additional point and added a subtle dig inline